aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -439,17 +437,36 @@ def parse_metrics_from_report_log(report_log_line):
     Returns:
         metrics - DatadogMetricPoint[]
     """
+
     regex_match = REPORT_LOG_REGEX.search(report_log_line)
 
     if not regex_match:
         return []
 
     metrics = []
-    tags = []
 
-    # loop is to account for adding cold start
-    for tag in TAGS_TO_PARSE_FROM_REPORT:
-        tags.append(tag + ":" + regex_match.group(tag))
+    tags = ["memorysize:" + regex_match.group(MEMORY_ALLOCATED_FIELD_NAME)]
+    if regex_match.group(INIT_DURATION_METRIC_NAME):
+        tags.append("cold_start:true")
+    else:
+        tags.append("cold_start:false")
+
+    # if cold_start:
+    if regex_match.group(INIT_DURATION_METRIC_NAME):
+        metric_point_value = float(regex_match.group(INIT_DURATION_METRIC_NAME))
+        # Multiply by 1/1000 to convert ms to seconds
+        metric_point_value *= METRIC_ADJUSTMENT_FACTORS[INIT_DURATION_METRIC_NAME]
+
+        initial_duration = DatadogMetricPoint(
+            "{}.{}".format(
+                ENHANCED_METRICS_NAMESPACE_PREFIX, INIT_DURATION_METRIC_NAME
+            ),
+            metric_point_value,
+        )
+
+        initial_duration.add_tags(tags)
+
+        metrics.append(initial_duration)
```

Comment:
@jcstorms1 my apologies, after a weekend, I totally forgot that another suggestion I pointed out last week was merging line 454-469 (after change) to the for loop of line 471 (after change), their logic are 99% same. I suggest adding `INIT_DURATION_METRIC_NAME` to `METRICS_TO_PARSE_FROM_REPORT` and add a check within the for loop like this

```python
for metric_name in METRICS_TO_PARSE_FROM_REPORT:
    if not regex_match.group(metric_name):
        continue
    metric_point_value = float(regex_match.group(metric_name))
    ....
```